update intro/icpc (#107)
OI-wiki/i18n: docs/intro/icpc.md

author: NachtgeistW

## Introduction

### ICPC

>  **ICPC**, International Collegiate Programming Contest, is organized by the ICPC Foundation. It is the most influential programming competition for college students. Since ACM sponsored this competition in the past, many people are used to calling it ACM competition.

Reviewer: ```suggestion
International Collegiate Programming Contest, or ICPC, is a competitive programming contest organized by the ICPC Foundation. It is the most influential programming competition for college students. Since ACM sponsored this competition in the past, many people are used to calling it ACM competition.
```

ICPC is mainly divided into two parts: Regionals and World Finals.

Official website: <https://icpc.baylor.edu> 

### CCPC

Official website: <https://ccpc.io> 

Chinese college student programming competition.

The major difference from ICPC is that many schools do not support reimbursement.

## Competition system introduction

> Contestants have multiple submission opportunities during the competition. The competition is evaluated in real time and the results are displayed (wrong test cases can be seen in some competitions, such as LeetCode). If the submitted result is wrong, there will be a penalty time. The more errors, the longer the penalty time. Each question can only get a score after all the points are correct. The competition ranking is judged based on the number of questions and penalty time.

Reviewer: solutions are judged in real time

There will be penalty if wrong solutions are submitted

```suggestion
Contestants have multiple submission opportunities during the competition. The competition is evaluated in real time and the results are displayed (wrong test cases can be seen in some competitions, such as LeetCode). If the submitted result is wrong, there will be a penalty time. The more errors, the longer the penalty time. The score of the problem can only be obtained after the submission is accepted. The ranking is judged based on the number of questions and penalty time.
```

In addition to ICPC and CCPC, many competitions also adopt this competition system, such as LeetCode Weekly Contest and National Programming Contest, Nowcoder Entry-level Practice Challenge and so on.

## Season schedule

- ICPC/CCPC Online Competition (from the end of August to the beginning of September)
- ICPC/CCPC Regional Competition (from the end of September to the end of November)
- ICPC EC Final/CCPC Final (Mid-December)

> - ICPC World Finals (April to June of the following year)

Reviewer: suggest:
Late August to Early September

## Training guide

### Multi-school joint training

> A training competition held in [HDU OJ](http://acm.hdu.edu.cn) in the summer. The competition has high quality questions and offers bonuses for winners. It has accumulated abundant resources throughout the years.

Reviewer: problem

> Keywords for query in OJ: `Multi-University Training Contest`.

Reviewer: search

### Domestic regional competition

> A collection of selected questions from OJs of universities in China can be found in [Virtual Judge](https://vjudge.net/).

Reviewer: problems

### Training Camp

> - Training camp organized by [Toutiao](https://en.wikipedia.org/wiki/Toutiao)/[Tsinghua University](https://www.tsinghua.edu.cn/en/index.htm)/CCPC (Wannafly Camp) during the winter vacation.

Reviewer: :+1: 可以 很 i18n
